autocompleter/base.py

```diff
@@ -273,16 +332,25 @@ def store(self, delete_old=True):
                 key = EXACT_SET_BASE_NAME % (provider_name,)
                 pipe.sadd(key, norm_term)
 
+        for facet in facet_dicts:
+            facet_value = facet['value']
+            facet_set_name = FACET_SET_BASE_NAME % (provider_name, facet_value,)
```

Comment:
just call this key... only because we do this every other place ... unless you want to change all those to be like terms_set_name exact_terms_set_name and so on